### DISCUSSION TITLE
Popper Bootstrap and Dropdown Menu not working

### DISCUSSION THREAD

**Discussion by luxora** [twbs/bootstrap#32079] [Show & Tell]
![overview](https://user-images.githubusercontent.com/9011593/98450530-a29f4280-213d-11eb-9e1b-2c4e99ce1b8d.png)
![includepopper](https://user-images.githubusercontent.com/9011593/98450533-a4690600-213d-11eb-92cf-c6cff2ab7ec4.png)

Could anyone tell me, how to fix this?
I would like to use Bootstraps Accordions with:
<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">

to get a kind pop out part for my website.
The tab called Über (about) is opened, should view some dropdown icons, but doesn't view them. The Dropdown Menu as using it standalone is working. The Accordions are working - using them standalone - too.

Furthermore I would like to use a regular Dropdown Menu Bar above, which should be responsive...
Therefore I included Bootstrap, Popper and Jquery as you can see on the Screen above.
The problem is, the menu got killed, or doesn't dropdown anymore...

I know there have been serveral persons asking for something like this, but I didn't see my combination with all of the above mentioned includes.

Thanks for your answer / help :)

**Comment by Deckluhm** [comment]
Can you provide the code? Pretty hard to help you with just an explanation of your problem and no code.

Also note that, as described [here ](https://getbootstrap.com/docs/4.5/getting-started/introduction/#js), you should put your JS files right before the `</body>` tag (that's probably not your issue here tho).

**Comment by luxora** [comment]
et voilà ... merci boucoup Deckuluhm!
Your link above and the code following below, helped me to fix the problem. (1)

I think, depending on which bootstrap file, which popper and which JS files and versions i'm including, the page gets another view or behaviour. For example with these ones below everything works. But when I'm including "older" Versions, sometimes my website won't be responsive anymore or killing the whole design? could that be possible? (2)

CODE:

If you decide to go with the separate scripts solution, Popper.js must come first, and then our JavaScript plugins.
`
<!DOCTYPE html>
<html>
<head>
<title>PAGE</title>
<?php
$urlhost = "http://www.mysite.de";
$subdir = "/web/";
$htmldir = $urlhost.$subdir."html/";
$imgdir = $urlhost.$subdir."img/";
$jsdir = $urlhost.$subdir."js/";
?>

 <meta name="viewport" content="width=device-width, initial-scale=1">


  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>


</head>
<body>
<div class="container">
<p> your code </p>
`

**Reply by Deckluhm** [reply]
> But when I'm including "older" Versions, sometimes my website won't be responsive anymore or killing the whole design? could that be possible?

I don't know about this specific issue but generally, you should always use the most recent version when possible because it comes with more bug fixes (and hopefully no new bugs :stuck_out_tongue:).

You can read version notes to know more about bug fixes and general changes (ex: [Bootstrap releases](https://github.com/twbs/bootstrap/releases)).
